### DISCUSSION TITLE
QUESTION: How to limit or order autocomplete results on columns only

### DISCUSSION THREAD

**Discussion by jancoow** [beekeeper-studio/beekeeper-studio#981] [Q&A]
**Details:**
Currently, when using autocomplete in for example an insert, it will list all possible PostgreSQL variables. How can I hide this from the autocomplete results? Or make sure that the columns are on top of the autocomplete list. 

**Screenshot / GIF [if appropriate]**:
![image](https://user-images.githubusercontent.com/9000749/145060275-ced31f30-e2a9-4271-9434-92960f1e62dc.png)

**Comment by rathboma** [comment]
Hey! Unfortunately I'm a bit hamstrung by the autocomplete library I'm currently using, which doesn't deal very well with context.

I'll definitely be reworking the autocomplete at some point in the future!
